muesli/web/viewsLecture.py

```diff
@@ -445,12 +445,16 @@ def emailTutors(request):
     db = request.db
     lecture = request.context.lecture
     form = LectureEmailTutors(request)
+    
     if request.method == 'POST' and form.processPostData(request.POST):
+        assistants_to_be_cced = []
+        if form['copytoassistants'] == 0:
+            assistants_to_be_cced = [assistant.email for assistant in lecture.assistants]
         tutors = lecture.tutors
         message = Message(subject=form['subject'],
                 sender=request.user.email,
                 to=[t.email for t in tutors],
-                cc=[assistant.email for assistant in lecture.assistants],
```

Comment:
Just for the future (as it obviously doesnt matter), you could have solved this using an `if`-Statement in the listcomprehension. 
```python
cc=[assistant.email for assistant in lecture.assistants if form['copytoassistants'] == 0],
``` 

Thank you very much for the contribution otherwise! :heart:

Reply:
Thanks for your suggestion. I've tested it and incorporated it into my PR.